app.go

```diff
@@ -108,26 +108,32 @@ func getHostName() string {
 	return name
 }
 
-func logger(conf Config) *logrus.Logger {
+func getServiceName(projectName, appName, envName string) string {
+	return fmt.Sprintf("%s-%s-%s", projectName, appName, envName)
+}
+
+func logger(conf Config) *logrus.Entry {
```

Comment:
What's the reason for switching to the `*logrus.Entry` instead of using the `*logrus.Logger`?

Reply:
We want to return logger.WithFields, which in a `*logrus.Entry`, so that all subsequent logging add the default fields